cmake/config.h.in

```diff
@@ -0,0 +1,43 @@
+#cmakedefine HAVE_FORK 1
+#cmakedefine HAVE_LSEEK64 1
+#cmakedefine HAVE_LUA 1
+#cmakedefine HAVE_POSIX_FADVISE 1
+#cmakedefine HAVE_POSIX_FALLOCATE 1
+#cmakedefine HAVE_PTHREAD 1
+#cmakedefine HAVE_SYNC_FILE_RANGE 1
+#cmakedefine HAVE_TERMIOS_H 1
+#cmakedefine HAVE_LIBGEN_H 1
+#cmakedefine HAVE_SYS_WAIT_H 1
+#cmakedefine HAVE_UNISTD_H 1
+#cmakedefine SIZEOF_OFF_T ${SIZEOF_OFF_T}
+
+#ifdef _MSC_VER
+#if _MSC_VER < 1900
+#define snprintf _snprintf
+#endif
+
+typedef __int64 ssize_t;
```

Comment:
Forgot about this, will add to PR tomorrow (maybe with building 32 bit on my pc)